administrator/components/com_localise/language/en-GB/com_localise.ini

```diff
@@ -630,3 +632,4 @@ COM_LOCALISE_TOOLTIP_TRANSLATIONS_COMPLETE_WITH_DEVELOP="Translation complete::%
 COM_LOCALISE_TOOLTIP_TRANSLATIONS_INPROGRESS_WITH_DEVELOP="Translation in progress<br>%1$s translated<br>%2$s unchanged<br>%3$s new strings<br>%4$s Modified values<br>%5$s total<br>%6$s extra"
 COM_LOCALISE_WARNING_DISABLED_ALLOW_DEVELOP_WITHOUT_LOCAL_SET="Note that the client %1$s is using a version of language files ('%2$s') different from the local installed instance ('%3$s') with 'Enable development reference language' disabled. Please, if you wanna continue with the feature disabled is suggested choose this client from the 'Location' filter to solve the issue."
 COM_LOCALISE_WRONG_LINE_CONTENT="Detected a parsing error within this file. The line <span style=\"background-color:red; color:black;\">%1$s</span> has not been added to the language file."
+COM_LOCALISE_ERROR_FILE_EMPTY="This reference file is empty: %s"
```

Comment:
please alpha order and change back to "This language file is empty: %s" as it can show the target language file.
Example: 
<img width="755" alt="Screen Shot 2021-11-03 at 07 27 38" src="https://user-images.githubusercontent.com/869724/140017522-f0842d57-8d8d-40f9-8bb9-40fb28d34f28.png">
and
<img width="738" alt="Screen Shot 2021-11-03 at 07 26 50" src="https://user-images.githubusercontent.com/869724/140017539-65f07cef-45b2-4a78-ba0a-cc01b60fd091.png">